.github/workflows/python-publish.yml

```diff
@@ -103,7 +103,7 @@ jobs:
             - linux/amd64
             - linux/arm64
           push: true
-          tags: ${{ env.IMAGE_NAME }}\:${{ needs.tagpr.outputs.tag }}
+          tags: ${{ env.IMAGE_NAME }}:${{ needs.tagpr.outputs.tag }}
```

Comment:
![Screenshot 2025-06-09 at 14 48 28](https://github.com/user-attachments/assets/383cb767-fa60-4155-ad02-f5a50ca94fe2)

![Screenshot 2025-06-09 at 14 48 23](https://github.com/user-attachments/assets/877a059d-7b9c-488b-853b-9a7debc22a99)

Reply:
When I see these color difference, I felt like I need to escape colon. However, it's not necessary, according to the log.